charts/argo-workflows/Chart.yaml

```diff
@@ -16,5 +16,5 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: added
-      description: Add support for executor args
+    - kind: fixed
+      description: Add HA docs and example
```

Comment:
This is now a docs and tests only change, so actually not sure if this necessary to bump anymore?

I used "fixed" because ArtifactHub doesn't support "docs" (["valid supported kinds are added, changed, deprecated, removed, fixed and security."](https://artifacthub.io/docs/topics/annotations/helm/))